clang/docs/ReleaseNotes.rst

```diff
@@ -59,6 +59,18 @@ C++ Specific Potentially Breaking Changes
 - Clang now performs semantic analysis for unary operators with dependent operands
   that are known to be of non-class non-enumeration type prior to instantiation.
 
+  This change uncovered a bug in libstdc++ 14.1.0 which may cause compile failures
+  on systems using that version of libstdc++ and Clang 19, with an error that looks
+  something like this:
+
+  .. code-block:: text
+
+    <source>:4:5: error: expression is not assignable
+    4 |     ++this;
+      |     ^ ~~~~
+
+  To fix this, update libstdc++ to a newer version, such as 14.1.1 or 14.2.
```

Comment:
```suggestion
  To fix this, update libstdc++ to version 14.1.1 or greater.
```
14.2 _should_ release before clang 19 but probably not by much so I don't think we want to be that specific